### ISSUE TITLE
Appnexus Keyword param broken in s2s setups

### ISSUE THREAD

**Issue by dbemiller** [prebid/Prebid.js#2513]
## Type of issue
Bug

## Description

See https://github.com/prebid/prebid-server/issues/499#issuecomment-387747141

If someone uses the Keywords param in theAppnexus adapter, it gets sent to Prebid Server like this:

```
"params": {
  "placementId": 12842089,
  "keywords": {
    "hp": 1,
    "in-view": 1,
    "in-focus": 1,
    "is-viewable": 1
  }
}
```  

The JSON schema for Appnexus in Prebid Server can be found here: https://github.com/prebid/prebid-server/blob/master/static/bidder-params/appnexus.json#L28

Basically we expect that `keywords` property to be:

```
{
  "keywords": [
    {
      "key": "hp",
      "value": ["1"]
    },
    {
      "key": "in-view",
      "value": ["1"]
    },
    ...
  ]
}
```

I wish I could fix this in Prebid Server... but Stored Requests make that a major breaking change. I think it needs to be fixed in Prebid.js instead.

**Comment by mkendall07** [comment]
Thanks Dave - we'll fix over here.

**Comment by jsnellbaker** [comment]
Duplicate to #2647, which is going to fixed in 1.14.0 of prebid.js (see #2667 for actual fix).
